### PR TITLE
fix(gooddata-pandas): Process total of grandtotals just once

### PR DESCRIPTION
**Pull request by torchbear**
JIRA: UNI-711

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-pandas/gooddata_pandas/result_convertor.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ class _AccumulatedData:
     grand_totals_headers: List[Optional[List[Dict[str, _DataHeaders]]]] = field(
         init=False, factory=lambda: [None, None]
     )
+    total_of_grant_totals_processed: bool = field(init=False, factory=lambda: False)
```
Comment (no23reason) [inline gooddata-pandas/gooddata_pandas/result_convertor.py:56]: Nitpick: instead of `factory=lambda: False` you should be able to do `default=False`.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-pandas/gooddata_pandas/result_convertor.py (hunk 1 above)
Comment (torchbear) [reply gooddata-pandas/gooddata_pandas/result_convertor.py]: fixed

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR512

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/296/ : FAILURE in 2m 25s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/446/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/450/ : SUCCESS in 11s

**Pull request merged**